src/router/routes/sign.js

```diff
@@ -140,6 +140,35 @@ const handleJob = async (job, clean = true) => {
   }
 };
 
+/* eslint-disable */
```

Comment:
Can you remove this API documentation? I've added the OpenAPI3 docs in doc/api.yaml. Lets keep it up-to-date as we go.